packages/cryptoassets/src/currencies.ts

```diff
@@ -1209,6 +1209,35 @@ const cryptocurrenciesById: Record<string, CryptoCurrency> = {
     ],
     explorerViews: [],
   },
+  helium: {
+    type: "CryptoCurrency",
+    id: "helium",
+    coinType: 904,
+    name: "Helium",
+    managerAppName: "Helium",
```

Comment:
I don't see any app called Helium at the moment in the Manager. cc @FabriceDautriat

Reply:
There should be one. Maybe only visible in developer mode.